webui/module/Client/src/Client/Model/ClientModel.php

```diff
@@ -114,6 +114,24 @@ public function getClientBackups(&$bsock=null, $client=null, $fileset=null, $ord
          }
          $result = $bsock->send_command($cmd, 2);
          $backups = \Zend\Json\Json::decode($result, \Zend\Json\Json::TYPE_ARRAY);
+
+         if(!isset($limit)) {
+            $filesets_pluginjob = array();
+            foreach ($backups['result']['backups'] as $key=>$backup) {
+               if (!array_key_exists($backup['fileset'], $filesets_pluginjob)) {
+                  $cmd = 'show fileset="'.$backup['fileset'].'"';
+                  $result = $bsock->send_command($cmd, 2);
+                  $fileset = \Zend\Json\Json::decode($result, \Zend\Json\Json::TYPE_ARRAY);
+                  $fileset_pluginjob[$backup['fileset']] = false;
+                  if (array_key_exists("plugin", $fileset['result']['filesets'][$backup['fileset']]['include'][0])) {
+                     if (!empty($fileset['result']['filesets'][$backup['fileset']]['include'][0]['plugin'])) {
+                        $fileset_pluginjob[$backup['fileset']] = true;
+                     }
+                  }
+               }
+               $backups['result']['backups'][$key]['pluginjob'] = $fileset_pluginjob[$backup['fileset']];
```

Comment:
```suggestion
               $backups['result']['backups'][$key]['pluginjob'] = $filesets_pluginjob[$backup['fileset']];
```

Reply:
Typo fixed in https://github.com/bareos/bareos/pull/1276/commits/f0cf31ae7030b95a0789d8d1f8a0d2c22188d22a .